### PR TITLE
ISPN-16235: Doc edits for topology set-stable command review

### PR DESCRIPTION
**Pull request by theashiot**
https://issues.redhat.com/browse/ISPN-16235

### CODE REVIEW

**Comment by theashiot** [comment]
Hi @Crumby,

Per our discussion in https://github.com/infinispan/infinispan/pull/12590 i've updated the text.

Can you please have a look?

thanks,
ashwin

File: documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@ However, we offer a CLI command to mark the current topology stable for a specif
 
 [IMPORTANT]
 ====
-Manually installing a topology can lead to data loss, only perform this operation if the initial topology cannot be recreated.
+Manually installing a topology can lead to data loss, only perform this operation if the initial topology cannot be recreated. The command cannot restore internal caches resulting in a partially incomplete cluster even when you restore the caches. There will be a loss of functionality for other caches requiring access to the missing internal caches.
```
Comment (jabolina) [inline documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc:9]: We could add a couple of examples. One internal cache is the cache for Protobuf schemas, so users are unable to upload schemas while there are missing nodes. The same applies to script execution/upload and the distributed locks.

Comment (theashiot) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc:9]: Thanks, @jabolina, how about this: 

The command cannot restore internal caches resulting in a partially incomplete cluster even when you restore the caches. There will be a loss of functionality for other caches requiring access to the missing internal caches. For example, there is one internal cache for Protobuf schemas. Users will be unable to upload schemas when there are missing nodes. Script execution , upload, and distributed locks are similarly affected.

Comment (jabolina) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: Sounds good to me. We can cut a few pieces:

The command does not work on internal caches. There will be a loss of functionality for caches requiring access to the internal caches with missing members. For example, users will be unable to upload schemas when there are missing nodes in the internal cache for Protobuf schemas. Script execution and upload, and distributed locks are similarly affected.

Let me know what you think.

Comment (sangeetaraghu) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: @jabolina @tristantarrant I am OOO and the writer @theashiot is also on PTO. Can either one of you can help add the minor edits suggested by @jabolina in the PR and merge it? That way this book will be ready for downstream product release. Thank you for your help and sorry for the inconvenience caused.

Comment (jabolina) [reply documentation/src/main/asciidoc/topics/proc_cli_set_topology_caches.adoc]: I can push the suggestions.

**Pull request merged**